### ISSUE TITLE
To use with type module (import)

### ISSUE THREAD

**Issue by ve3** [follow-redirects/follow-redirects#225]
```
import http from 'follow-redirects';
import https from 'follow-redirects';

https.request(options);
http.request(options);
```

Returns

> https.request is not a function
> http.request is not a function

Changed to `import * as https from 'follow-redirects';`  also the same.

Changed to `import {https} from 'follow-redirects';` show errors:

> SyntaxError: Named export 'https' not found.

So, I have to change those import to:

```
import http from 'follow-redirects/http.js';
import https from 'follow-redirects/https.js';
```

And now everything works fine.

I hope that this maybe note in the readme file because more and more people using `import` module. 😀

**Comment by dickorydock** [comment]
Thank you for posting this! Do you know if there is a way to adapt the use of require() in this post?:  https://github.com/follow-redirects/follow-redirects/issues/110

**Comment by ve3** [comment]
Which `require()`? I see a lot of them I don't know which one.

**Comment by dickorydock** [comment]
For example a replacement for ` http: require('http'),` in here: https://github.com/follow-redirects/follow-redirects/issues/110#issuecomment-493786878

**Comment by ve3** [comment]
It is already in my post above.
`import http from 'follow-redirects/http.js';`

**Comment by dickorydock** [comment]
Ah yeah, I tried dropping that into the .wrap() statement but I just have to define it earlier and then use http in the .wrap() statement. Thanks!

**Comment by ve3** [comment]
Yes, `import` must be on top. But I have no idea how to put it in `wrap()`.

**Comment by ve3** [comment]
@dickorydock Try this.

```lang-javascript
import nodehttp from 'node:http';
import nodehttps from 'node:https';
import followRedir from 'follow-redirects';

const {http, https} = followRedir.wrap({
    http: nodehttp,
    https: nodehttps,
})

const URLTarget = 'https://google.com';
https.get(URLTarget, (response) => {
    console.log('response', response);
})
.on('error', (error) => {
    console.error(error);
});
```

**Comment by RubenVerborgh** [comment]
Thanks @ve3. At the moment, support for `import` is not part of the API, so the above tricks might or might not keep on working. The current method is `import fr from 'follow-redirects'` and take it from there; we should add module support though.
